doc/api/api_changes.rst

```diff
@@ -35,6 +33,6 @@ added to Matplotlib, see :ref:`whats-new`.
       :glob:
       :maxdepth: 1
 
-      next_api_changes/*
+      the most recent api_changes_X.Y/*
```

Comment:
```suggestion
      api_changes_3.3/*
```
For now. This defines toc entries and thus must point to existing files. We probably should think of how to handle this page in the future.